src/archivematicaCommon/lib/archivematicaFunctions.py

```diff
@@ -75,9 +75,24 @@ def unicodeToStr(string):
         string = string.encode("utf-8")
     return string
 
-def strToUnicode(string):
+def strToUnicode(string, obstinate=False):
     if isinstance(string, str):
-        string = string.decode("utf-8")
+        if obstinate:
+            # Obstinately try to get a Unicode instance: attempt to decode to
+            # using the 'ascii', 'utf8', and finally 'latin-1' codecs. If all
+            # fail, decode using 'ascii', ignoring indecipherable bytes.
+            try:
+                string = string.decode('ascii')
```

Comment:
Given that ASCII is a subset of UTF8, I think we can probably skip this attempt.